### ISSUE TITLE
Pixelated font rendering

### ISSUE THREAD

**Issue by matthias-vogt** [felixhageloh/uebersicht#233]
Font rendering is a bit pixelated by default right now.
I tried changing `-webkit-font-smoothing` and that fixed it.

`-webkit-font-smoothing: subpixel-antialiased;` (default right now)
![subpixel-antialiased](https://cloud.githubusercontent.com/assets/4030513/18753673/6371c6a6-80e6-11e6-86e2-2d8c57e1eb99.png)

`-webkit-font-smoothing: antialiased;`
![antialiased](https://cloud.githubusercontent.com/assets/4030513/18753674/63725ff8-80e6-11e6-8e64-08deacd45c7e.png)

Maybe you could consider making `-webkit-font-smoothing: antialiased;` the default.

**Comment by felixhageloh** [comment]
Hi! I also always use `antialiased` for my widgets, but what strikes me as odd is that `subpixel-antialiased` looks more pixelated (usually it looks smooth, but heavier with letter shapes slightly distorted). Are you seeing this on retina or non-retina display?

**Comment by matthias-vogt** [comment]
On a retina display. The weird thing to me is that this is not an issue in safari, even though it's using the same webkit distribution (right?)

**Comment by felixhageloh** [comment]
yeah exactly - I'll research a bit and see what the best default is then

**Comment by felixhageloh** [comment]
(i'm considering to just remove this rule and see what how the default looks)

**Comment by matthias-vogt** [comment]
I just noticed #17 exists, sorry for the dupe
